linera-core/src/local_node.rs

```diff
@@ -155,22 +171,29 @@ where
     S: Storage,
     ViewError: From<S::StoreError>,
 {
+    #[tracing::instrument(level = "trace", skip(state))]
```

Comment:
Nit: same here (and I guess everywhere else? :sweat_smile:)

```suggestion
    #[tracing::instrument(level = "trace", skip_all)]
```